app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,34 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
 
+   <fragment
+       android:id="@+id/mainContainer"
+       android:name="androidx.navigation.fragment.NavHostFragment"
+       android:layout_width="0dp"
+       android:layout_height="0dp"
+       app:defaultNavHost="true"
+       app:layout_constraintBottom_toTopOf="@+id/bottomNavigationView"
+
+       app:layout_constraintEnd_toEndOf="parent"
+       app:layout_constraintHorizontal_bias="0.0"
+       app:layout_constraintStart_toStartOf="parent"
+       app:layout_constraintTop_toTopOf="parent"
+       app:layout_constraintVertical_bias="0.0"
+       app:navGraph="@navigation/nav_graph" />
+
+   <com.google.android.material.bottomnavigation.BottomNavigationView
+       android:id="@+id/bottomNavigationView"
+       android:layout_width="350dp"
+       android:layout_height="65dp"
+       android:layout_marginBottom="16dp"
+       android:background="@drawable/round_edge"
+       app:itemIconTint="@color/black"
+       app:itemTextColor="#000000"
```

Comment:
please remove these colors from here to enable active/inactive tab colors from default BottomNavigationView implementation. In the future we will update the colors if we want, but for now let's go in this way